progenitor-impl/src/method.rs

```diff
@@ -1418,7 +1418,7 @@ impl Generator {
             }
         });
 
-        let maybe_clone = cloneable.then(|| quote! { #[derive(Clone)] });
+        let maybe_clone = cloneable.then(|| quote! { #[derive(Clone, Debug)] });
```

Comment:
Is Clone required here, or was it just convenient to lump these together? If it's the latter, I'd suggest something like...

```rust
let mut derives = vec![quote!{ Debug }];
if cloneable {
    derives.push(quote!{ Clone });
}

let derive = quote!{
    #[derive( #( #derives ),* )]
};
```
```

Reply:
I like your approach much more, but it looks like I won't be able to use it unfortunately :( I got this error, and I'm not sure how to fix it?
```console
error[E0277]: the trait bound `std::vec::Vec<TokenStream2>: ToTokens` is not satisfied
    --> progenitor-impl/src/method.rs:1490:12
     |
1490 |           Ok(quote! {
     |  ____________^
1491 | |             #[doc = #struct_doc]
1492 | |             #derives
1493 | |             pub struct #struct_ident<'a> {
...    |
1535 | |             }
1536 | |         })
     | |_________^ the trait `ToTokens` is not implemented for `std::vec::Vec<TokenStream2>`
     |
     = help: the following other types implement trait `ToTokens`:
               &'a T
               &'a mut T
               Abi
               Abstract
               AndAnd
               AndEq
               AngleBracketedGenericArguments
               Arm
             and 308 others
     = note: this error originates in the macro `$crate::quote_token_with_context` (in Nightly builds, run with -Z macro-backtrace for more info)
```

Reply:
I'm not sure of the problem without seeing the code, but you may need `derive` rather than `derives` if you're using the code I suggested above.

Reply:
🤦‍♀️ hahahah yep, I was using `derives` instead of `derive`

Reply:
... it's my own fault for my uncreative naming..